Lib/fontTools/varLib/varStore.py

```diff
@@ -95,7 +95,8 @@ def storeMasters(self, master_values, *, round=round):
         return base, self.storeDeltas(deltas, round=noRound)
 
     def storeDeltas(self, deltas, *, round=round):
-        deltas = [round(d) for d in deltas]
+        if round != noRound:
```

Comment:
why this? if round == noRound then it's no-op, right? not sure if this micro-optimization is worth it

Reply:
Sure I can drop that.